src/components/Select.js

```diff
@@ -313,18 +347,16 @@ const Select = ({ children, id, label, required, placeholder, onChange, fallback
 	);
 };
 
-
 //Make Option available as a SubComponent
 // <Select.Option>....</Select.Option>
-Select.Option = Option
-
+Select.Option = Option;
 
 Select.defaultProps = {
 	readOnly: false,
 	id: null,
 	required: false,
-	placeholder: "Select an option",
-	fallback: "No result found"
-}
+	placeholder: 'Choisir une option',
+	fallback: 'No result found',
```

Comment:
Should you change the fallback message too ?

Reply:
true